sktime/datatypes/tests/test_utils.py

```diff
@@ -46,7 +48,9 @@ def test_get_cutoff(scitype, mtype, return_index):
         if fixture is None:
             continue
 
-        cutoff = get_cutoff(fixture, return_index=return_index)
+        cutoff = get_cutoff(
```

Comment:
`check_input` and `convert_input` are not tested

Reply:
how would you test them?

`convert_input` is just there to avoid unnecessary conversions in case the user already knows it is in one of the internal types, so the only way to test what it does would be to look into what lines are being executed?

`check_input` raises errors for incompatible input, so I could write a negative test.

Reply:
> Do all of these cases pass without errors? Do we get the same cutoff regardless?

The case where it makes a difference is if the object has a type not internally supported. If conversions are turned off, the function crashes (as the docstring tells you)

Reply:
done, please look

Reply:
great! I am sure we will all be thankful for these tests later in the future :)